ai_sperm_analysis/utils/casa_metrics.py

```
@@ -0,0 +1,350 @@
+            'vap': vap,
+            'lin': lin,
+            'str': str_val,
+            'wob': wob,
+            'alh': alh,
+            'bcf': bcf,
+            'pattern': pattern_type,
+            'quality_score': quality_score,
+            'who_grade': self.get_who_grade(vsl, lin)
+        }
+    
+    def classify_movement_pattern(self, vcl, vsl, lin, str_val, alh):
```

Ruff (0.12.2)

Unused method argument: `str_val`

(ARG002)